### PR TITLE
created a file

### PR DESCRIPTION
**Pull request by isaka-lumato**
hello microverse project 

image

The aim of the project was to make us learn some useful tools that will help through the course and eventually career, includes;

- GitHub flow, branches
- Linting and how to set it up
- Correctly writing ReadMe file, commit messages, and pull request summary

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi @isaka-lumato   👋
### This a good start! 👍 
You've followed the Github flow rules. ✔️ 
Your linter reports no errors ✔️ 

However, there are some changes that you are required to make before this project can be approved.

## Required Changes ♻️
- [x] You currently do not have a custom README. It is important to have a descriptive README because it is the first file a person will see when they encounter your project. You can use the Microverse README template.
- [x] You currently do not have a Pull Request (PR) summary. You're expected to have a PR summary that contains a short description of what was achieved/changes made in this PR.


## Optional Changes 👌
_OPTIONAL change/requirements are not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

- [ ] It is advisable that your commit messages be capitalized, and written in the imperative tense. You can watch [this video](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6) on how to edit your commit messages using `git rebase`.
Examples of a good commit message:
 i. Add HTML files to github
ii. Add CSS files to github

#### If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

### Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#), however, if you feel this review is unfair, you can request a second opinion using this [form](https://airtable.com/shrQAqnBwek5a0O0s).

## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# hello-world
\ No newline at end of file
+Hello microverse
+
+its my first project in microverse
+
+
+the aim is to learn the github basics and other microverse professional standards
```
Comment (Temmarie) [inline README.md]: - [ ] The project should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
A good README should have a description that is clear, short, and to the point. It should describe the importance of your project, what it does and how to use it. This way technical and non-technical users can understand how to use it.  Refer to this template [here](https://github.com/microverseinc/readme-template) and customise it to your needs.

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle:

### Hi @isaka-lumato       ! :wave: :wave:

#### Good job so far!
You have implemented most of the required changes!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
- [ ] Kindly use the provided Template for your readme file. 

_Check the comments under the review._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 1 code review left in this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi @isaka-lumato ! :wave: :wave:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ OPTIONAL] The project should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
A good README should have a description that is clear, short, and to the point. It should describe the importance of your project, what it does and how to use it. This way technical and non-technical users can understand how to use it.  Refer to this template [here](https://github.com/microverseinc/readme-template) and customise it to your needs. **Note that while this is optional, it is a vital part of every project. **


#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianotieno — COMMENTED** [review]
Hi @isaka-lumato , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**